freud/interface.py

```diff
@@ -47,15 +44,9 @@ def compute(self, system, query_points, neighbors=None):
                 <https://freud.readthedocs.io/en/stable/topics/querying.html>`_
                 (Default value: None).
         """  # noqa E501
-        cdef:
-            freud.locality.NeighborQuery nq
-            freud.locality.NeighborList nlist
-            freud.locality._QueryArgs qargs
-            const float[:, ::1] l_query_points
-            unsigned int num_query_points
 
-        nlist = freud.locality._make_default_nlist(
-            system, neighbors, query_points)
+        # freud.locality.
+        nlist = _make_default_nlist(system, neighbors, query_points)
```

Comment:
```suggestion
        nlist = _make_default_nlist(system, neighbors, query_points)
```
That comment was not doing anything.